Utilities/Flex/Model/TransientTokenGenerator/TransientTokenGenerator.cs

```diff
@@ -0,0 +1,81 @@
+namespace CyberSource.Utilities.Flex.Model.TransientTokenGenerator
+{
+    using System;
+    using System.Collections.Generic;
+
+    using System.Globalization;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+
+    public partial class TransientTokenGenerator
```

Comment:
The name should **NOT** be `TransientTokenGenerator`. Change this to `TransientTokenModel`

Reply:
Also, you do **NOT** need this class to be in its own folder. Move it to `Utilities/Flex/Model` and change the namespace.